datalad/support/gitrepo.py

```diff
@@ -2284,10 +2359,27 @@ def update_remote(self, name=None, verbose=False):
             expect_stderr=True
         )
 
-    # TODO: centralize all the c&p code in fetch, pull, push
-    # TODO: document **kwargs passed to gitpython
+    def fetch(self, remote=None, refspec=None, all_=False, git_options=None,
+              **kwargs):
+        """
+        kwargs :
+          Deprecated. GitPython-style keyword argument for git-push.
+          Will be appended to any git_options.
+        """
```

Comment:
For consistency, perhaps the docstrings for this and the other new `x/x_` pairs follow what `add/add_`:  full doc-string for the former, "Like `x`, but returns a generator" for the latter.

Reply:
Ok, will do.

Reply:
Done.